.gitignore

```diff
@@ -26,7 +26,6 @@ devel/
 downloads/
 eggs/
 .eggs/
-lib/
```

Comment:
Could be entirely wrong, but would it be better to use the .gitignore in ./client directory to specifically allow that referenced !lib/? instead of allowing it in the root for all folders?  I imagine there is a way to do this, or there could be some other reason I don't about.  If we don't want to have a  contradictory rule in a sub folder (I understand how that is nasty), perhaps a specific allowance for the ./client/src/lib folder in the root).

Reply:
I could be wrong, but disallowing `lib/` in the first place doesn't apply to any of the code or build dist/packaging in this project.  It's a little heavy-handed to block all folders called lib, I think.